### DISCUSSION TITLE
Error invalid dynamic link domain (action code settings)

### DISCUSSION THREAD

**Discussion by ANIBIT14** [invertase/react-native-firebase#4259] [Q&A] [answered]
For implementing Email Link Sign In when I am defining action code settings with dynamicLinkDomain key and passing the needed dynamic link url to it, the following error is being thrown -:
[Error: [auth/invalid-dynamic-link-domain] The provided dynamic link domain is not configured or authorized for the current project.]

but when I am not using the above key it picks up the default dynamic link and everything's works, but as our project ha multiple apps I needed the dynamic link url of my choice to use.

Hope anybody can tell how to resolve the above error

**Comment by jordynchuhaloff** [comment, marked as answer]
In your Firebase console, go to Authentication -> Sign-in method, scroll down to Authorized domains. Make sure you've added your `dynamicLinkDomain` to that list.

**Reply by Carl4** [reply]
Looks like this has moved to `Authentication` -> `Settings `-> `Authorized Domains`, however I get this error for a domain that is on the list.

**Edit**: Never mind about the error. I was using the deprecated dynamic_link_domain parameter.
